tools/cli/commands/generate.js

```diff
@@ -446,6 +446,10 @@ function addMirrorRepo( composerJson, name, org, autotagger ) {
 	composerJson.extra.changelogger[
 		'link-template'
 	] = `https://github.com/${ org }/${ name }/compare/v\${old}...v\${new}`;
+	// Handle cases where we need more autotagger info for github action project types.
+	if ( autotagger && name.match( /^action-/ ) ) {
```

Comment:
This works, but it's too bad we can't directly look at whether the user selected "github-action".

Reply:
Yeah, we'd have to pass that down pretty far. I could look at reworking this all if you think that would be better, but might be more trouble than it's worth. 

...also maybe I was happy to flex some regex skills I've picked up along the way.

Reply:
I don't think it's worth the trouble.

Sorry, I have a habit of complaining about things even though I'm ok with leaving them how they are. (: Maybe I shouldn't have assumed that approving would be enough of a signal that that was the case.

Reply:
> Maybe I shouldn't have assumed that approving would be enough of a signal that that was the case.

No worries! Now I know :)